paasta_tools/deployd/metrics.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import time
+
+from paasta_tools.deployd.common import PaastaThread
+
+try:
+    import yelp_meteorite
```

Comment:
A more friendly way to do this is to "just" use python-statsd directly instead of the yelp_meterorite wrapper, except we don't get all the "free" dimensions :(